M/Mineos/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Mineos"
+version = v"1.0"
+
+# Collection of sources required to build Mineos
+sources = [
+    "https://github.com/anowacki/mineos.git" =>
+    "e2558b486d7656ef112608a8776643da66dc87cf",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mineos
+./configure --prefix=$prefix --host=$target --disable-doc
+make
+make install
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
+    MacOS(:x86_64),
+    FreeBSD(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
+platforms = expand_gcc_versions(platforms)
```

Comment:
Thanks for your contribution!

I see that you're using an older version of BinaryBuilder, here in Yggdrasil we use the `master` version of the package, which has a slightly different syntax.  This is not a problem, we'll help you to adjust the builder.

Why do you need this?

Reply:
Thank you very much for your help!

> Why do you need this?

I'm not sure quite what you mean, but if you mean 'why do you need this artifact', then it is so that I can distribute a package which relies on Mineos (not yet public); if you mean 'why do you need these platforms', then I don't and your later suggestion of using `supported_platforms()` is what I need to change to.

Reply:
I updated the comment after writing: why did you need `expand_gcc_versions`?  Is the program in C++ or Fortran?

Reply:
Yes, it is mainly Fortran with some C.

Reply:
Ok, then set `platforms` to
```julia
platforms = expand_gfortran_versions(supported_platforms())
```

Reply:
Thanks very much.  For some reason I thought the documentation was out of date here as I couldn't find an `expand_gfortran_versions()` function; this will have been an error on my part.  I think I was fooled as my local builds seemed to work.

Reply:
This function is in the new `BinaryBuilder`: https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.expand_gfortran_versions-Tuple{Platform}.

OK, the build machine is building, let's see how it goes!